test/Header/SetCookieTest.php

```diff
@@ -343,6 +343,12 @@ public function testDoesNotAcceptCookieNameFromArbitraryLocationInHeaderValue()
         $this->assertNotEquals('leo_auth_token', $setCookieHeader->getName());
     }
 
+    public function testEmptyExpiresSection()
+    {
+        $cookie = 'Set-Cookie:path=/; Expires=; HttpOnly;';
+        $setCookieHeader = SetCookie::fromString($cookie);
```

Comment:
There is no assertion, so nothing is tested. You can use a try/catch block to test if an exception is thrown. Also please update the method name so it reflects what you are actually testing.

You can do it like this:
```php
public function testEmptyExpiresDoesNotThrowException()
{
    $cookieString = 'Set-Cookie:path=/; Expires=; HttpOnly;';

    try {
        SetCookie::fromString($cookie);
    } catch (\Exception $exception) {
        $this->fail('Empty "expires" value should not throw an exception');
    }
    
    $this->pass('No exception was thrown for empty "expires" value');
}
```